backend/src/modules/doubles-tournament/core/domain/MatchesQueue.ts

```diff
@@ -0,0 +1,12 @@
+import { QueuedMatch } from './QueuedMatch';
+import { TournamentId } from './TournamentId';
+
+export class MatchesQueue {
```

Comment:
Troche mi nie pasuje, że to jest w oddzielnym pliku. Albo QueuedMatch daj tez do oddzielnego i wtedy jakos nazwij same funckej, albo wszystko do MatchesQueue.

Reply:
QueuedMatch jest przecież osobnym obiektem domenowym. Wiele razy też tworzę zarówno MatchesQueue jak i QueuedMatch, więc mi na dwa obiekty domenowe to jakoś gra,
i teraz jest raptem jedna metoda w pliku QueuedMatch - pushMatchToQueue i już :-)